### PR TITLE
Update copy doc url - managed/kubernetes

### PR DESCRIPTION
**Pull request by WendyS1000**
## DONE 
I have updated the copy doc url for this web page. 

## QA 
Make sure the copy doc url link is correct. 

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/5317

### CODE REVIEW

File: templates/kubernetes/managed.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {% block title %}Managed Kubernetes | Kubernetes{% endblock %}
 {% block meta_description %}Canonical is the leading managed private cloud provider. We will build, support and manage your Kubernetes private cloud for only $5-15 per server per day.{% endblock meta_description %}
-{% block meta_copydoc %}https://docs.google.com/document/d/15jzhasokjH5Ql9Za56ZxRknXNNhEXYdIFXXtLODGxns/edit#heading=h.q9m33avd7enn{% endblock meta_copydoc %}
+{% block meta_copydoc %}https://docs.google.com/document/d/15jzhasokjH5Ql9Za56ZxRknXNNhEXYdIFXXtLODGxns/edit#heading=h.sq5ginkhkome{% endblock meta_copydoc %}
```
Comment (anthonydillon) [inline templates/kubernetes/managed.html]:
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/15jzhasokjH5Ql9Za56ZxRknXNNhEXYdIFXXtLODGxns{% endblock meta_copydoc %}
```

Comment (anthonydillon) [reply templates/kubernetes/managed.html]: This should work.

**Comment by SirSamTumless** [comment]
This looks good to me!

**Comment by anthonydillon** [comment]
The title of this copydoc says draft. Is this the correct one?

**Comment by anthonydillon** [comment]
:+1:

**Pull request merged**